ClientTests/NavigationPathTests.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+#if FOCUS
+@testable import Firefox_Focus
+#else
+@testable import Firefox_Klar
+#endif
+
+class NavigationPathTests: XCTestCase {
+    
+    private var appScheme: String {
+        AppInfo.isKlar ? "firefox-klar" : "firefox-focus"
+    }
+    
+    private var appURL: String!
+    private var navigation: NavigationPath!
+    private let url = "https://www.apple.com/"
+    private let badURL = "boomer"
+    
+    override func setUp() {
+        super.setUp()
+        appURL = ""
```

Comment:
`appURL` can be a local variable for each test

Reply:
Same for `navigation`. Usually we call it `sut` (system under test) to imply what we're testing